tests/json/schema-1-response/input.py

```diff
@@ -0,0 +1,124 @@
+#! test QC_JSON Schema for properties
+
+import numpy as np
+import psi4
+import json
+
+# Generate JSON data
+json_data = {
+  "schema_name": "qcschema_input",
+  "schema_version": 1,
+  "molecule": {
+    "geometry": [
+      0.0,
+      0.0,
+      -0.1294769411935893,
+      0.0,
+      -1.494187339479985,
+      1.0274465079245698,
+      0.0,
+      1.494187339479985,
+      1.0274465079245698
+    ],
+    "symbols": [
+      "O",
+      "H",
+      "H"
+    ],
+    "fix_com": True,
+    "fix_orientation": True
+  },
+  "driver": "properties",
+  "model": {
+    "method": "CC2",
+    "basis": "6-31G",
+  },
+  "keywords": {"scf_type": "df",
+               "mp2_type": "df",
+               "e_convergence": 9,
+               "omega": [355,439,'nm'],
+               "gauge": "velocity",
+               "function_kwargs": { "properties": [
+                                    "dipole",
+                                    "polarizability",
+                                    "rotation",
+                                    "roa_tensor"
+                                ]
+               }
+  }
+}
+                            
+import copy
+json_data_copy = copy.deepcopy(json_data)
+
+# Write expected output (dipole & quadrupole in au)
+
+expected_response = {
+   'CC2 DIPOLE POLARIZABILITY @ 355NM': 5.175390333179149,
+   'CC2 DIPOLE POLARIZABILITY @ 439NM': 5.063282247200858,
+   'CC2 DIPOLE X': 0.0,
+   'CC2 DIPOLE Y': 0.0,
+   'CC2 DIPOLE Z': 2.5171476216528084,
+   'CC2 QUADRUPOLE XX': -7.393278028527795,
+   'CC2 QUADRUPOLE XY': -1.8387569175060344e-16,
+   'CC2 QUADRUPOLE XZ': 0.0,
+   'CC2 QUADRUPOLE YY': -4.3142280005261755,
+   'CC2 QUADRUPOLE YZ': 0.0,
+   'CC2 QUADRUPOLE ZZ': -5.9972667929848065,
+   'CC2 SPECIFIC ROTATION (MVG) @ 355NM': -0.0,
+   'CC2 SPECIFIC ROTATION (MVG) @ 439NM': -0.0,
+   'CC2 SPECIFIC ROTATION (VEL) @ 355NM': -0.0,
+   'CC2 SPECIFIC ROTATION (VEL) @ 439NM': -0.0,
+   'CC2 DIPOLE': np.array([0, 0, 0.99032208]),
+   'CC2 QUADRUPOLE': np.array([[-5.49672082e+00, -1.36707065e-16,  0.00000000e+00],
```

Comment:
Is this how `yapf` formats those?

Reply:
ah, I forgot about `yapf`. formatted and pushed the change, thanks!